### PR TITLE
MAUI-871860-[Others]-Prepare the UG documentation for the unbound row support

### PR DESCRIPTION
**Pull request by sudarsanMS**
### Description
Prepare the UG documentation for the unbound row support

### CODE REVIEW

File: MAUI/DataGrid/overview.md [hunk 1]
```diff
@@ -47,4 +47,6 @@ The .NET MAUI DataGrid control is used to display and manipulate data in a tabul
 
 * **Exporting** - Support to export the data to Excel and PDF.
 
+ **Unbound Row** - It allows to add additional rows at top and also bottom of the SfDataGrid which are not bound with data object from underlying data source.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/overview.md:50]: It allows adding rows at the top and bottom of the DataGrid which are not bound with data objects from the underlying data source.

File: MAUI/DataGrid/styling.md [hunk 2]
```diff
@@ -287,6 +287,44 @@ The group summary row can be customized by the writing style for [DataGridGroupS
 
 ![Group summary row styling in .NET MAUI DataGrid](Images/styling/maui-datagrid-group-summary-row-style.png)
 
+### Styling Unbound row cell
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/styling.md:290]: Styling unbound row cell

File: MAUI/DataGrid/styling.md [hunk 3]
```diff
@@ -287,6 +287,44 @@ The group summary row can be customized by the writing style for [DataGridGroupS
...
+            <Setter Property="FontFamily" Value="TimesNewRoman"/>
+        </Style>
+    </ContentPage.Resources>
+</ContentPage>
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Unbound row cell styling in .NET MAUI DataGrid" src="Images\styling\maui-datagrid-unboundRowCellStyling.png" width="404"/>
+
+### Styling Unbound row
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/styling.md:311]: Styling unbound row

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]
As we discussed, change the following,

1. Provide the code snippets with selection mode as single while populating the data to the unbound row.
2. Provide the image name properly.
3. Remove the unnecessary namespaces.
4. Include the missing topics compared to Xamarin UG

File: MAUI/DataGrid/overview.md (hunk 1 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/overview.md:50]: Modified

File: MAUI/DataGrid/styling.md (hunk 2 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/styling.md:290]: edited

File: MAUI/DataGrid/styling.md (hunk 3 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/styling.md:311]: edited

File: MAUI/DataGrid/unbound-row.md [hunk 4]
```diff
@@ -229,4 +220,14 @@ private void DataGrid_QueryRowHeight(object? sender, DataGridQueryRowHeightEvent
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unboundRowHeight.png" width="404"/>
+<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unbound-row-height.png" width="404"/>
+
+## Get unbound rows
+
+You can get the unbound row at the specified row index using [GetUnboundRow]() method.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/unbound-row.md:227]: The unbound row can be obtained by using the `GetUnboundRowAtRowIndex` method.

File: MAUI/DataGrid/unbound-row.md [hunk 5]
```diff
@@ -229,4 +220,14 @@ private void DataGrid_QueryRowHeight(object? sender, DataGridQueryRowHeightEvent
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unboundRowHeight.png" width="404"/>
+<img alt="Changing unbound row height" src="Images\unboundRow\maui-datagrid-unbound-row-height.png" width="404"/>
+
+## Get unbound rows
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/unbound-row.md:225]: Get unbound row

File: MAUI/DataGrid/unbound-row.md [hunk 6]
```diff
@@ -116,60 +117,51 @@ private void Datagrid_SelectionChanged(object? sender, DataGridSelectionChangedE
...
+datagrid.SelectionChanged += Datagrid_SelectionChanged;
+private void Datagrid_SelectionChanged(object? sender, DataGridSelectionChangedEventArgs e)
+{
+    datagrid!.InvalidateUnboundRow(datagrid!.UnboundRows[0]);
 }
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="Populating data for unbound rows" src="Images\unboundRow\maui-datagrid-unboundRowPopulationData.png" width="404"/>
+<img alt="Populating data for unbound rows" src="Images\unboundRow\maui-datagrid-unbound-row-refreshing.png" width="404"/>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/unbound-row.md:164]: Image is not required. Remove the image.

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

File: MAUI/DataGrid/unbound-row.md (hunk 4 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/unbound-row.md:227]: Added

File: MAUI/DataGrid/unbound-row.md (hunk 5 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/unbound-row.md:225]: modified

File: MAUI/DataGrid/unbound-row.md (hunk 6 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/unbound-row.md:164]: removed

**Pull request merged**